### ISSUE TITLE
Relevance of a criterion for 3.5 Ensure Your Solutions Are Accessible

### ISSUE THREAD

**Issue by ldevernay** [w3c/sustyweb#58]
I might be missing something but I don't see how carbon-aware design relates to Accessibility.
https://w3c.github.io/sustyweb/#success-criterion-carbon-aware-design

**Comment by AlexDawsonUK** [comment]
I think when it was added the benefit was looking from an ESG approach (the social awareness) but I agree it's probably a bit too abstract, especially as this is explicitly a guideline about carbon, not about ESG. I'll remove it for the next draft.

**Comment by AlexDawsonUK** [comment]
I have made the change to the specification, this will be reflected in the next draft.
